docs/src/data/links.ts

```diff
@@ -0,0 +1,134 @@
+export interface ComponentNavItem {
+  href: string;
+  label: string;
+  body: string;
+}
+
+const sortByLabel = (a: ComponentNavItem, b: ComponentNavItem) =>
+  a.label < b.label ? -1 : a.label > b.label ? 1 : 0;
+
+export const baseComponents: ComponentNavItem[] = [
+  {
+    href: '/components/view',
+    label: 'View',
+    body: `An Alert displays a brief, important message in a way that attracts the user's attention without interrupting the user's task. Alerts are typically intended to be read out dynamically by a screen reader.`,
+  },
+  {
+    href: '/components/text',
+    label: 'Text',
+    body: `The Text primitive is used to display simple strings of text in an interface.`,
+  },
+  {
+    href: '/components/heading',
+    label: 'Heading',
+    body: ``,
+  },
+  {
+    href: '/components/icon',
+    label: 'Icon',
+    body: ``,
+  },
+  {
+    href: '/components/image',
+    label: 'Image',
+    body: ``,
+  },
+  {
+    href: '/components/divider',
+    label: 'Divider',
+    body: ``,
+  },
+].sort(sortByLabel);
+
+export const connectedComponents = [
+  {
+    href: '/components/authenticator',
+    label: 'Authenticator',
+    body: 'The Authenticator component adds complete authentication flows to your application with minimal boilerplate.',
+  },
+  {
+    href: '/components/chatbot',
+    label: 'Chatbot',
+    body: 'Chatbot automatically renders a complete chat messaging interface that can be used out-of-the-box, or it can be customized using theming support.',
+  },
+  {
+    href: '/components/s3album',
+    label: 'S3 Album',
+    body: 'Renders a list of S3Image objects.',
+  },
+].sort(sortByLabel);
+
+export const dataDisplayComponents = [
+  { href: '/components/badge', label: 'Badge', body: `` },
+  { href: '/components/rating', label: 'Rating', body: `` },
+].sort(sortByLabel);
```

Comment:
Is there any performance worry here? It is probably nothing, but should we just sort these by hand in the source?

Reply:
Since the docs are SSG/SSR, I don't think there's much performance impact.